lib/diff.ml

```diff
@@ -113,17 +116,21 @@ let rec items ~reference ~current =
   let env = Typing_env.for_diff ~reference ~current in
   let ref_items = extract_items reference in
   let curr_items = extract_items current in
-  Sig_item_map.merge
-    (fun (item_type, name) ref_opt curr_opt ->
-      match (item_type, ref_opt, curr_opt) with
-      | Value_item, reference, current ->
-          value_item ~typing_env:env ~name ~reference ~current
-      | Module_item, reference, current ->
-          module_item ~typing_env:env ~name ~reference ~current
-      | Type_item, reference, current ->
-          type_item ~typing_env:env ~name ~reference ~current)
-    ref_items curr_items
-  |> Sig_item_map.bindings |> List.map snd
+  let values_diff = [] in
+  let modules_diff = [] in
+  let types_diff = [] in
```

Comment:
You won't need 3 distinct calls here, just one that will return all diffs. The fact that we have different maps is an implementation detail, from the outside we just access a single data structure holding all type of items.